src/components/TopNav.js

```diff
@@ -0,0 +1,23 @@
+import '../css/TopNav.css';
+import { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+import { FaHatWizard } from 'react-icons/fa';
+
+const TopNav = () => {
+  return (
+    <header>
+      <h1>Squizard's Spells</h1>
+      <div className="top-nav-btns-box">
+        <Link className="top-nav-btn" to="/">
+          Home
+        </Link>
+        <Link className="top-nav-btn" to="/my-spellbook">
+          <FaHatWizard className="icon-small" aria-hidden="true"/>
+          My Spellbook
+        </Link>
+      </div>
```

Comment:
I wonder if these buttons should be `<NavLink></NavLink>` instead?  I think that the home button could go away when we are on the landing page.  Either that or we could style it like a browser tab?  Where it is highlighted when the tab is active?

Something like this?
![tab style navigation](https://newtodesign.com/wp-content/uploads/2018/09/Modern-Tabs-Design-with-pure-CSS.png)

Reply:
I love the tab look. I did it for a previous project, and it turned out to be surprisingly difficult, but I think that was mostly because I wanted each tab and the page it was connected to to be a different color. 
I think it could be a good solution here though. 
Because we don't need the home button when we're home, and ideally the My Spellbook button would change to a Browse Spells button when we are on the spellbook page. Doing the tabs might be simpler than trying to make it different on each page.